SPECS/python-rpm-spec/python-rpm-spec.spec

```
@@ -0,0 +1,56 @@
+# sitelib for noarch packages, sitearch for others (remove the unneeded one)
+%{!?__python2: %global __python2 %__python}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%{!?python2_sitearch: %global python2_sitearch %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
+
+%if 0%{?fedora}
+%bcond_without python3
+%else
+%bcond_with python3
+%endif
+
+Name:           python-rpm-spec
+Version:        0.14.1
+Release:        1%{?dist}
+Summary:        A Python library for parsing RPM spec files.
+
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/bkircher/python-rpm-spec
+#Source0:       https://github.com/bkircher/python-rpm-spec/archive/refs/tags/v0.14.1.tar.gz
+Source0:        python-rpm-spec-v0.14.1.tar.gz   
```

Please use single-line source tags and upload the source in the `%{name}-%{version}.tar.gz` format:
```suggestion
Source0:       https://github.com/bkircher/python-rpm-spec/archive/refs/tags/v0.14.1.tar.gz#/%{name}-%{version}.tar.gz
```